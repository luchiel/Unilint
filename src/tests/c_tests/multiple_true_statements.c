int main()
{
    if(a) printf("bad"); else printf("bad");
    while(a) printf("bad");
    for(int i = 0; i < 1; ++i) printf("bad");
}