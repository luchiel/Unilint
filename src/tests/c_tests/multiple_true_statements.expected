3:11: two or more expressions at one line
3:26: two or more expressions at one line
3:31: two or more expressions at one line
4:14: two or more expressions at one line
5:32: two or more expressions at one line
5:46: two or more expressions at one line
